README.md

```diff
@@ -154,111 +154,163 @@ Here is a list of the versions of the wheels and libraries that are included in
-  * The commit before we removed fork. A few packages don't build if fork is not defined.
-* libcxx: wasix-org/llvm-project 6bb93a243f6d15855f485f5aec3810d9e2de150d
-  * This is the commit that is currently used by wasix-libc
-* compiler-rt: wasix-org/llvm-project 6bb93a243f6d15855f485f5aec3810d9e2de150d
-  * This is the commit that is currently used by wasix-libc
-<!-- LIB_VERSIONS_END -->
+- libzbar: 0.23.93
+- libffi: wasix-org/libffi main
+- pandoc: haskell-wasm/pandoc wasm
+- postgresql: 17.5
+- brotli: 1.1.0
+- zlib: develop
+  - 1.3.1 does not have proper cmake support, so we are using develop for now
+- libjpeg-turbo: 3.1.1
+- xz: 5.8.1
+- libtiff: 4.7.0
+- libwebp: 1.5.0
+- giflib: 5.2.2
+- libpng: 1.6.50
+- SDL: 3.2.16
+  - SDL has all subsystems disabled
+- openjpeg: 2.5.3
+- libuv: 1.51.0
+- mariadb-connector-c: 3.4.6
+- openssl: 3.5.1
+- bzip2: 1.0.8
+- util-linux: 2.41.1
+  - We only build libuuid from util-linux
+- openssh: 10.0p2
+- dropbear: 2025.88
+- tinyxml2: 11.0.0
+- geos: 3.13.1
+- libxslt: 1.1.43
+- libxml2: 2.14.5
+- google-crc32c: 1.1.2
+- arrow: 19.0.1
+- arrow: 21.0.0
+- rapidjson: 1.1.0
+- icu: 77.1
+- readline: 8.2
+- ncurses: 6.4.20230225
+- curl: curl/curl ab18c04218ff316cd67b1e928c5cee579b2f66a0
+  - This was the current commit in the wasix fork. We can probably update to the next release
+- pycurl: 7.45.6
+- sqlite: 3.50.4
+- wasix-libc: wasix-org/wasix-libc e4e5cfcf1009e690207821e9b3a63cdce2ee9e86
+  - The commit before we removed fork. A few packages don't build if fork is not defined.
+- libcxx: wasix-org/llvm-project 6bb93a243f6d15855f485f5aec3810d9e2de150d
+  - This is the commit that is currently used by wasix-libc
+- compiler-rt: wasix-org/llvm-project 6bb93a243f6d15855f485f5aec3810d9e2de150d
+  - This is the commit that is currently used by wasix-libc
+  <!-- LIB_VERSIONS_END -->
+
+## Tests
+
+The whole point of these tests is to ensure that the packages works within the wasmer runtime.
+So there are two ways of running the tests, natively and with wasmer:
+
+### Native
+
+Requirements:
+
+- Python 3.10+
+- `python -m venv env`
+- `source venv/bin/activate`
+- `pip install .`
+- `pip install pytest`
+
+I'm sure it's possible to do via uv, or poetry, or many other ways. But this works also.
+
+Run tests:
+
+- `python -m pytest`
+- Discovers files matching `*-test.py`, `*_test.py`, and `test_*.py` under `tests/`.
+- Files ending with `.skip.py` are ignored.
+- Files ending with `-broken.py` are marked as expected failures (strict). If they pass, the run reports XPASS and fails.
+
+### Wasmer
+
+These are the instructions as of 2025-10, but there's a shipit looming, which may optimize the flow.
+
+Requirements:
+
+- wasmer 6.1.0-rc.3+
+
+Run tests:
+
+- `uv pip compile pyproject.toml --python-version=3.13 --universal --extra-index-url https://pythonindex.wasix.org/simple --index-url=https://pypi.org/simple --emit-index-url --only-binary :all: -o wasmer-requirements.txt`
```

Comment:
I think you may be able to convert the test script to a PEP 723 script with URL based specifiers. That way we should be able to just `uv run` the tests.